.eslintrc

```diff
@@ -1,3 +1,5 @@
+extends: eslint:recommended
```

Comment:
So, do we want me to remove the 5 duplicated rules, or are we cool to merge as-is?